flow-typed/npm-custom/array-move_v1.x.x.js

```diff
@@ -0,0 +1,8 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Moving them all into `flow-typed/npm-custom` should prevent us from mistakenly overwriting them again, and I locked them into the semver versions.